migrate.js

```diff
@@ -24,6 +24,7 @@ async function migrate (opts) {
 }
 
 const defaults = {
+  arcVersion: '0.0.1-rc.32',
```

Comment:
how about taking the default from the installed packaged (`node_module/...`)

Reply:
This is also an issue as suppose you just used an older Arc, then it will immediately become the new default instead of the latest.

Reply:
though you mention is specifically at https://github.com/daostack/migration/blob/master/package.json#L46

Reply:
But it changes there as well when you're running the script with a new version.